src/main/java/org/jabref/gui/preview/PreviewViewer.java

```diff
@@ -144,7 +144,7 @@ private void highlightSearchPattern() {
                     "var markInstance = new Mark(document.getElementById(\"content\"));" +
                             "markInstance.unmark({" +
                             "  done: function(){" +
-                            "    markInstance.markRegExp(/" + pattern + "/gmi);" +
+                            "    markInstance.markRegExp(/" + pattern.replaceAll("/", "\\\\/") + "/gmi);" +
```

Comment:
Since `/` terminates the JavaScript regexp, it should be escaped. The issue with using `new RegExp` is that you will have to escape the string terminator instead, so it is more code without any benefit that I am aware of.